### PR TITLE
feat: Organizer Menu Bar: Simplified Settings and merged Export Page Areas

### LINKED ISSUE #6011: Organizer Menu Bar: Simplify Settings and merge Export Page Areas
We already have a lot of features, but some features are not so regularly used and can move a bit to the back. This includes the "Export" feature, which also needs to be tested more in detail. 

To simplify this please **merge the Export and Settings page** as follows:
* On the settings page implement a left side menu similar to the menu on the tickets tab with
   * **Export** - Simply move the content of the export page to the subpage here and use the URL `/settings/export` and delete the export page and menu item in the event menu

compare screenshot
Screenshot from 2020-12-12 14-38-02

   * **Options** - Show the heading "Danger Zone" here and add the current info of the settings page using the URL `/settings/options`

Also check how Github is doing it in Repository Settings if you need to understand more details about this approach.

Compare left side bar in tickets tab
Screenshot from 2020-12-12 14-26-57

### PR DESCRIPTION
**Pull request by Afzal-Ind**
Fixes #6011 

#### Short description of what this resolves:
Merged Export and settings page like Ticket's tab

#### Changes proposed in this pull request:

- Modified existing files
- Added new files
- See this changes here [After](https://user-images.githubusercontent.com/11625672/102022286-d0664f80-3dab-11eb-8d33-df3362bc9a8a.JPG)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Afzal-Ind** [comment]
Plz review it. Thanks

File: config/environment.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = function(environment) {
     appName                  : process.env.APP_NAME || 'Open Event',
     modulePrefix             : 'open-event-frontend',
     environment,
-    rootURL                  : process.env.ROOT_URL || '/',
+    rootURL                  : process.env.ROOT_URL || 'open-event-frontend',
```
Comment (sachinchauhan2889) [inline config/environment.js:16]: not needed to get changed.

File: package.json [hunk 2]
```diff
@@ -81,6 +81,7 @@
     "ember-cli-dotenv": "^3.1.0",
     "ember-cli-fastboot": "^2.2.3",
     "ember-cli-flash": "^2.0.0",
+    "ember-cli-github-pages": "^0.2.2",
```
Comment (sachinchauhan2889) [inline package.json:84]: no need of ember-cli-github-pages for this issue to get solved.

File: yarn.lock [hunk 3]
```diff
@@ -6636,6 +6636,14 @@ ember-cli-get-component-path-option@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/ember-cli-get-component-path-option/-/ember-cli-get-component-path-option-1.0.0.tgz#0d7b595559e2f9050abed804f1d8eff1b08bc771"
 
+ember-cli-github-pages@^0.2.2:
```
Comment (sachinchauhan2889) [inline yarn.lock:6639]: not needed.

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
can't complete my review as deployment is not opening.
this error is poping up. 
new review

review new console

**Comment by Afzal-Ind** [comment]
> can't complete my review as deployment is not opening.
> this error is poping up.
> new review
> 
> review new console

It's Alright. You can take time.

**Comment by sachinchauhan2889** [comment]
> > can't complete my review as deployment is not opening.
> > this error is poping up.
> > new review
> > review new console
> 
> It's Alright. You can take time.

is deployment opening at your side????
All deployment are opening in my side. i think you need to do changes in your code as reviewers are not able to reach deployment.

File: yarn.lock (hunk 3 above)
Comment (Afzal-Ind) [reply yarn.lock:6639]: Hi... Can I delete this file?

Comment (maze-runnar) [reply yarn.lock:6639]: wait...what? are u saying to remove yarn file? 
>Hi... Can I remove this file?

NO

Comment (Afzal-Ind) [reply yarn.lock:6639]: > wait...what? are u saying to remove yarn file?

Oh sorry, my mistake. I am kind of naive here.

**Comment by iamareebjamal** [comment]
Please write proper PR title

**Review by daretobedifferent18 — COMMENTED** [review]
Screenshot from 2020-12-14 23-50-27
Pls by default preselect export tab
Presently blank is coming

**Comment by Afzal-Ind** [comment]
> Pls by default preselect export tab


Thanks, I will try to do that. So other than this, is everything ok?

**Review by daretobedifferent18 — COMMENTED** [review]
Screenshot from 2020-12-15 00-37-30
Unneccesary space on right of tabbednavigation

**Comment by Afzal-Ind** [comment]
> Please write proper PR title

Changed

**Comment by Afzal-Ind** [comment]
> Unneccesary space on right of tabbednavigation

I don't see any unnecessary space here. Plz look here 
c3

**Review by daretobedifferent18 — COMMENTED** [review]
https://open-event-frontend-mb5gp0fqy.vercel.app/events/c1b86687/settings/export
Pla go to this and have a look of uneccessary space

**Comment by Afzal-Ind** [comment]
> https://open-event-frontend-mb5gp0fqy.vercel.app/events/c1b86687/settings/export

Sorry, it didn't work from my side.

**Comment by Afzal-Ind** [comment]
> Presently blank is coming

It's fixed now. U can recheck. Thanks to @sachinchauhan2889 .

**Comment by Afzal-Ind** [comment]
Plz review it. Thanks. @iamareebjamal ,

**Comment by Afzal-Ind** [comment]
> plz @iamareebjamal.

**Review by daretobedifferent18 — COMMENTED** [review]
Screenshot from 2020-12-19 22-56-33
I don't know but still I'm getting this blank space and due to this other styling is not looking gud like boxes size became small

**Comment by Afzal-Ind** [comment]
> I don't know but still I'm getting this blank space

Plz explore ticket tab, it's kind of replica of ticket nav tabbed. Thanks.

**Comment by daretobedifferent18** [comment]
> > I don't know but still I'm getting this blank space
> 
> Plz explore ticket tab, it's kind of replica of ticket nav tabbed. Thanks.

Thats working absolutely fine

**Comment by Afzal-Ind** [comment]
> Thats working absolutely fine

Then plz tell what I need to do.. Becaz I just merged export page on settings tab as per requirement.

File: app/templates/events/view/settings/export.hbs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <div class="ui stackable very relaxed container grid">
-  <div class="eight wide column">
+  <div class="six wide column">
```
Comment (daretobedifferent18) [inline app/templates/events/view/settings/export.hbs:2]: why

File: app/templates/events/view/settings/export.hbs [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <div class="ui divider"></div>
     <Events::View::Export::DownloadCommon @downloadType="iCalendar XML (xCal)" @downloadUrl="calendar.xcs" @model={{this.model}} />
   </div>
-  <div class="eight wide column">
```
Comment (daretobedifferent18) [inline app/templates/events/view/settings/export.hbs:11]: why

File: app/templates/events/view/settings/export.hbs (hunk 4 above)
Comment (Afzal-Ind) [reply app/templates/events/view/settings/export.hbs:2]: yep I undo!

Comment (Afzal-Ind) [reply app/templates/events/view/settings/export.hbs]: Done. Plz check.

**Comment by Afzal-Ind** [comment]
Plz do review here, thanks.

**Comment by iamareebjamal** [comment]
@sachinchauhan2889 @maze-runnar Please review

File: app/controllers/events/view/settings/options.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 
-export default class extends Controller {
+export default class OptionsController extends Controller {
```
Comment (sachinchauhan2889) [inline app/controllers/events/view/settings/options.js:4]: use _danger-zone.js_ as file name.  rename file.

File: app/router.js [hunk 7]
```diff
@@ -79,8 +79,10 @@ Router.map(function() {
         this.route('sessions-speakers');
         this.route('attendee');
       });
-      this.route('export');
-      this.route('settings');
+      this.route('settings', function() {
+        this.route('export');
+        this.route('options');
```
Comment (sachinchauhan2889) [inline app/router.js:84]: danger-zone

File: app/templates/events/view/settings/options.hbs [hunk 8]
```diff
@@ -0,0 +1,58 @@
+<div class="row">
```
Comment (sachinchauhan2889) [inline app/templates/events/view/settings/options.hbs:1]: rename file name danger-zone.hbs

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
## fix this, in mobile view put some space between tab options and page content.
new review 21 dec


## in desktop view, remove space between tab and export page. have a look at danger-zone content.
new review 21 dec space

File: app/router.js (hunk 7 above)
Comment (maze-runnar) [inline app/router.js:84]: I think rename file to danger-zone but keep route to options.
`Options - Show the heading "Danger Zone" here and add the current info of the settings page using the URL /settings/options`

File: app/templates/events/view/settings/export.hbs [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<div class="row">
```
Comment (sachinchauhan2889) [inline app/templates/events/view/settings/export.hbs:1]: i think this is not needed

File: app/templates/events/view/settings/export.hbs [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+ <div class="ui stackable very relaxed container grid">
```
Comment (sachinchauhan2889) [inline app/templates/events/view/settings/export.hbs:2]: for removing space between tab and export page, try to remove "very relaxed container" class

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]

File: app/templates/events/view/settings/export.hbs (hunk 10 above)
Comment (Afzal-Ind) [reply app/templates/events/view/settings/export.hbs:2]: Thanks for this suggestion that I wanted to have.

File: app/templates/events/view/settings/options.hbs (hunk 8 above)
Comment (Afzal-Ind) [reply app/templates/events/view/settings/options.hbs:1]: I think not required as per issue's requirement

**Comment by Afzal-Ind** [comment]
> ## please, fix this, in mobile view put some space between tab options and page content.

I think you can see here the comparison between ticket tab and setting tab, the same space is put here.
 
Screenshot_20201221-121254

Screenshot_20201221-121220

File: app/templates/events/view/settings/export.hbs [hunk 11]
```diff
@@ -1,16 +1,14 @@
...
-     <div class="ui divider"></div>
-     <Events::View::Export::DownloadCommon @downloadType="iCalendar XML (xCal)" @downloadUrl="calendar.xcs" @model={{this.model}} />
-   </div>
-   <div class="eight wide column">
-     <Events::View::Export::ApiResponse @eventId={{this.model.id}} />
-   </div>
- </div>
-</div>
\ No newline at end of file
+<div class="ui stackable">
```
Comment (sachinchauhan2889) [inline app/templates/events/view/settings/export.hbs:1]: "grid" class is required because we are using "column" class below.

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]

**Review by sachinchauhan2889 — APPROVED** [review]

**Review by sachinchauhan2889 — APPROVED** [review]

**Comment by sachinchauhan2889** [comment]
looks good to me. 
@iamareebjamal sir, this PR is ready for your review.

**Pull request merged**